DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: rbabylon
 Type: Package
 Title: R package for babylon
-Version: 0.6.1.8001
+Version: 0.6.1.7002
```

Comment:
Do we want to decrement the version?

Reply:
This is kind of a grey area in our versioning scheme. Technically yes. We saw to increment this dev version number when you _open_ the branch, and then pull it in when you merge to `develop` so that the version number in `develop` always reflects the last branch that was pulled. But yes, sometimes this does mean we decrement it. 

It doesn't bother me, but if you have strong feelings I'm open.

Reply:
I'm fine with leaving it as is.